src/ReactCrop.tsx

```diff
@@ -423,8 +425,25 @@ class ReactCrop extends PureComponent<ReactCropProps, ReactCropState> {
     (this.componentRef.current as HTMLDivElement).focus({ preventScroll: true }); // All other browsers
 
     const rect = (this.mediaWrapperRef.current as HTMLDivElement).getBoundingClientRect();
-    const x = (e.clientX - rect.left) / zoom;
-    const y = (e.clientY - rect.top) / zoom;
+    let x = 0;
+    let y = 0;
+    let scaledX = (e.clientX - rect.left) / zoom;
+    let scaledY = (e.clientY - rect.top) / zoom;
+    let degrees = spin;
+    let radians = Math.abs((degrees * Math.PI) / 180.0);
+    if ((degrees > -45 && degrees <= 45) || (Math.abs(degrees) > 135 && Math.abs(degrees) <= 180)) {
+      // Top and Bottom
+      x = scaledX * Math.cos(radians);
```

Comment:
So this is the condition it normally enters and there is no difference between the previous output (`x = (e.clientX - rect.left) / zoom;`) if spin is 0?

Reply:
I will merge and check later anyway just curious

Reply:
Since `Math.cos(0) = 1`, when the spin is 0 that will be evaluated as `x = scaledX * 1`, so there won't be any difference.